src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -205,6 +205,23 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
 
         return viewableSessions;
     }
+    
+    /**
+     * checks if there is any email sent from the course
+     * @param courseId
+     * @param userEmail
+     * @return true if there is any email sent from the course.
+     */
+    public boolean isAnyEmailSentForTheCourse(String courseId) {
+        List<FeedbackSessionAttributes> sessions = getFeedbackSessionsForCourse(courseId);
+
+        for (FeedbackSessionAttributes session : sessions) {
+            if (session.isSentOpenEmail() || session.isSentPublishedEmail()) {
```

Comment:
I saw this routine somewhere before. Can extract this to a small method in the `FeedbackSessionAttributes` class, like:
```java
public boolean isSentRelevantEmails() { // change the method name as needed
    return isSentOpenEmail() && isSentPublishedEmail();
}
```